src/Illuminate/Support/helpers.php

```diff
@@ -172,6 +172,20 @@ function array_has($array, $keys)
     }
 }
 
+if (! function_exists('array_keys_exits')) {
```

Comment:
Should be `array_keys_exist`, no?

Reply:
According to my knowledge of the english language `exists` is the simple present tense. It would be great if more native english speakers can help us out with this :)

Reply:
If you're referring to a singular object's existence, `exists` would be correct (This chair exists).

When referring to multiple objects, `exist` is the correct form (These chairs exist).

Reply:
Oh! Thanks @36864 🙂